scripts/installer

```diff
@@ -435,7 +435,8 @@ install() {
 # uninstall invokes kubectl delete with the built manifest.
 uninstall() {
   echo "Uninstalling ..."
-  kubectl delete -f $TMP_FILE
+  kubectl api-resources --verbs=list -o name | \
```

Comment:
Should we use `--verbs=list,delete` since for example `componentstatuses` cannot be deleted?

Reply:
Yeah, maybe delete is enough.

Reply:
oops that's what I meant 😆 👍

Reply:
fixed